llvm/test/Transforms/SCCP/ipscp-drop-argmemonly.ll

```diff
@@ -11,10 +11,10 @@
 ; Here the pointer argument %arg will be replaced by a constant. We need to
 ; drop argmemonly.
 ;.
-; CHECK: @[[G:[a-zA-Z0-9_$"\\.-]+]] = internal global i32 0
+; CHECK: @g = internal global i32 0
 ;.
 define internal void @ptrarg.1(ptr %arg, i32 %val) argmemonly nounwind {
-; CHECK: Function Attrs: nounwind memory(readwrite, inaccessiblemem: none)
+; CHECK: Function Attrs: nounwind memory(readwrite, inaccessiblemem: none, errnomem: none)
```

Comment:
I wonder if we should be setting errnomem as well in https://github.com/llvm/llvm-project/blob/64735ad63975c1126f84dbf7921ce4341dfa2419/llvm/lib/Transforms/IPO/SCCP.cpp#L194-L195. The propagated global might be errno.

Reply:
That's true, added this as well.